src/main/scala/com/exasol/cloudetl/kinesis/KinesisShardDataImporter.scala

```diff
@@ -20,23 +20,19 @@ object KinesisShardDataImporter {
     val region = kinesisArgumentsReader.getRegionArgument
     val shardId = kinesisArgumentsReader.getShardIdArgument
     val shardSequenceNumber = kinesisArgumentsReader.getShardSequenceNumber
-    val records = importRecordsForShard(
-      awsAccessKeyId,
-      awsSecretAccessKey,
-      awsSessionToken,
-      streamName,
-      region,
-      shardId,
-      shardSequenceNumber
-    )
+    val awsCredentials =
+      new BasicSessionCredentials(awsAccessKeyId, awsSecretAccessKey, awsSessionToken)
+    val amazonKinesis = AmazonKinesisClientBuilder.standard
+      .withRegion(region)
+      .withCredentials(new AWSStaticCredentialsProvider(awsCredentials))
+      .build
```

Comment:
Add this to kinesis properties class as a method that return kinesis client, e.g, `val kinesisClient = kinesisConsumerProperties.createClient()` or `.buildClient()`.

That way, you can remove variables required for kinesis client.

Reply:
done